htdocs/contact/class/contact.class.php

```diff
@@ -1189,8 +1189,7 @@ public function findNearest($id = 0, $lastname = '', $firstname = '', $ref_ext =
 		// Then search on email
 		// Then search on lastname + firstname
 		// Then search ref_ext or alias with a OR
-		$tmpcontact = new Contact($this->db);
-		$result = $tmpcontact->fetch($id, null, $ref_ext, $email, 0, $socid);
+		$result = $this->fetch($id, null, $ref_ext, $email, 0, $socid);
```

Comment:
This looks strange because the function should return only the ID found with no need to load data.
Loading of data is done later with $thirdpartystatic->fetch($result); line 2672 of emailcollector.class.php